Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs

```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
+          propertyStub);
+      var validator = new EqualValidator(
+          5,
+          validationMessage);
+
+      var instanceToValidate = new object();
+      var context = new PropertyValidatorContext(
+          new ValidationContext(instanceToValidate),
+          instanceToValidate,
+          propertyStub,
+          3);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain("5"));
+      }
+    }
+
+    [Test]
+    [TestCaseSource(nameof(cultures))]
+    public void ExclusiveRangeValidator_CanBuildFormattedValidationMessage (string culture)
+    {
+      var factory = new LocalizedValidationMessageFactory(SafeServiceLocator.Current.GetInstance<IGlobalizationService>());
+
+      var propertyStub = new NullPropertyInformation();
+      var validationMessage = factory.CreateValidationMessageForPropertyValidator(
+          new ExclusiveRangeValidator(5, 7, new InvariantValidationMessage("fake")),
+          propertyStub);
+      var validator = new ExclusiveRangeValidator(
+          2,
+          4,
+          validationMessage);
 
+      var instanceToValidate = new object();
+      var context = new PropertyValidatorContext(
+          new ValidationContext(instanceToValidate),
+          instanceToValidate,
+          propertyStub,
+          4);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@"2.*4"));
+      }
+    }
+
+    [Test]
+    [TestCaseSource(nameof(cultures))]
+    public void InclusiveRangeValidator_CanBuildFormattedValidationMessage (string culture)
+    {
+      var factory = new LocalizedValidationMessageFactory(SafeServiceLocator.Current.GetInstance<IGlobalizationService>());
+
+      var propertyStub = new NullPropertyInformation();
+      var validationMessage = factory.CreateValidationMessageForPropertyValidator(
+          new InclusiveRangeValidator(5, 7, new InvariantValidationMessage("fake")),
+          propertyStub);
+      var validator = new InclusiveRangeValidator(
+          2,
+          4,
+          validationMessage);
+
+      var instanceToValidate = new object();
+      var context = new PropertyValidatorContext(
+          new ValidationContext(instanceToValidate),
+          instanceToValidate,
+          propertyStub,
+          1);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@"2.*4"));
```

Comment:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@" 2 .* 4 "));
```
If this results in green test, too.